content/doc/administration/requirements/java.adoc

```diff
@@ -7,27 +7,32 @@ There are separate run and job execution requirements for Jenkins installations.
 
 ## Running Jenkins
 
-Java 8 or Java 11 are required for running modern versions of Jenkins. Upgrading an existing Jenkins setup from Java 8 to Java 11? See the link:/doc/administration/requirements/upgrade-java-guidelines[upgrade guidelines].
+Jenkins requires Java 11 or 17 since Jenkins 2.357 and LTS 2.361.1.
+link:/blog/2022/06/28/require-java-11/[Read more about this in the announcement blog post.]
 
-Java 11 Docker installation instructions are included in link:/doc/book/installing/docker/#downloading-and-running-jenkins-in-docker["Downloading and running Jenkins in Docker"]
+[NOTE]
+.upgrading Java version to 11
+====
+Upgrading an existing Jenkins setup from Java 8 to Java 11?
+Refer to the link:/doc/administration/requirements/upgrade-java-guidelines[upgrade guidelines].
+====
 
-Starting with the Jenkins 2.357 weekly release, Java 11 or Java 17 will be required.
-Java 8 is supported on the https://www.jenkins.io/download/lts/[Long Term Support] release line until the September 2022 LTS release.
+## Running Jenkins on Java 11 in Docker
 
-Java 17 users can run the `jenkins/jenkins:jdk17` Docker image.
+Java 11 Docker installation instructions are included in link:/doc/book/installing/docker/#downloading-and-running-jenkins-in-docker["Downloading and running Jenkins in Docker"].
+Alternatively, the `jenkins/jenkins:jdk17` Docker image allows you to run the Jenkins controller on Java 17.
 
 All other Java versions are not supported.
 
-These requirements apply to all components of the Jenkins system, including Jenkins controller,
-all types of agents, CLI clients, and other components.
+These requirements apply to all components of the Jenkins system, including the Jenkins controller, all types of agents, CLI clients, and other components.
 
-Jenkins project performs a full test flow with the following JDK/JREs:
+The Jenkins project performs a full test flow with the following JDK/JREs:
 
 * OpenJDK JDK / JRE 11 - 64 bits
 * OpenJDK JDK / JRE 17 - 64 bits
 
 JRE/JDKs from other vendors are supported and may be used.
-See link:/redirect/issue-tracker[our Issue tracker] for known Java compatibility issues.
+Refer to link:link:https://issues.jenkins.io/issues/?jql=labels%3Djdk[our issue tracker] for known Java compatibility issues.
```

Comment:
Only need one copy of the word 'link'
```suggestion
Refer to link:https://issues.jenkins.io/issues/?jql=labels%3Djdk[our issue tracker] for known Java compatibility issues.
```

Reply:
Ah good catch @MarkEWaite, committing that suggestion now